<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>260</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableView{
	border: 2px solid gray;
	border-radius: 5px;
}</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QGroupBox" name="dialpad_groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>250</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox#dialpad_groupBox{
	background-color: white;
	border: 2px solid gray;
    border-radius: 5px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>220</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    border: transparent;
    border-radius: 20px;
    border-style: outset;
    background-color: rgba(218,218,218,200);
}</string>
     </property>
     <property name="text">
      <string>8
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    border: transparent;
    border-radius: 20px;
    border-style: outset;
    background-color: rgba(218,218,218,200);
}</string>
     </property>
     <property name="text">
      <string>2
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>160</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    border: transparent;
    border-radius: 20px;
    border-style: outset;
    background-color: rgba(218,218,218,200);
}</string>
     </property>
     <property name="text">
      <string>6
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    border: transparent;
    border-radius: 20px;
    border-style: outset;
    background-color: rgba(218,218,218,200);
}</string>
     </property>
     <property name="text">
      <string>7
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_back">
     <property name="geometry">
      <rect>
       <x>167</x>
       <y>347</y>
       <width>30</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Reference Sans Serif</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    border: transparent;
    border-radius: 20px;
    border-style: outset;
    background-color: rgba(218,218,218,200);
}</string>
     </property>
     <property name="text">
      <string>4
</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="phoneNum_lineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>15</y>
       <width>200</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_0">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>280</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    border: transparent;
    border-radius: 20px;
    border-style: outset;
    background-color: rgba(218,218,218,200);
}</string>
     </property>
     <property name="text">
      <string>0
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_asterisk">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    border: transparent;
    border-radius: 20px;
    border-style: outset;
    background-color: rgba(218,218,218,200);
}</string>
     </property>
     <property name="text">
      <string>*</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>160</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    border: transparent;
    border-radius: 20px;
    border-style: outset;
    background-color: rgba(218,218,218,200);
}</string>
     </property>
     <property name="text">
      <string>5
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_hashtag">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>280</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    border: transparent;
    border-radius: 20px;
    border-style: outset;
    background-color: rgba(218,218,218,200);
}</string>
     </property>
     <property name="text">
      <string>#</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    border: transparent;
    border-radius: 20px;
    border-style: outset;
    background-color: rgba(218,218,218,200);
}</string>
     </property>
     <property name="text">
      <string>3
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_call">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>340</y>
       <width>40</width>
       <height>38</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>45</width>
       <height>45</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>220</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    border: transparent;
    border-radius: 20px;
    border-style: outset;
    background-color: rgba(218,218,218,200);
}</string>
     </property>
     <property name="text">
      <string>9
</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    border: transparent;
    border-radius: 20px;
    border-style: outset;
    background-color: rgba(218,218,218,200);
}</string>
     </property>
     <property name="text">
      <string>1
</string>
     </property>
     <property name="iconSize">
      <size>
       <width>45</width>
       <height>45</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="addNumber_label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>62</y>
       <width>100</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#addNumber_label{
	color: #0779e4
}</string>
     </property>
     <property name="text">
      <string>Add Number</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="abc_label">
     <property name="geometry">
      <rect>
       <x>113</x>
       <y>120</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>ABC</string>
     </property>
    </widget>
    <widget class="QLabel" name="def_label">
     <property name="geometry">
      <rect>
       <x>172</x>
       <y>120</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>DEF</string>
     </property>
    </widget>
    <widget class="QLabel" name="ghi_label">
     <property name="geometry">
      <rect>
       <x>52</x>
       <y>180</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>GHI</string>
     </property>
    </widget>
    <widget class="QLabel" name="jkl_label">
     <property name="geometry">
      <rect>
       <x>113</x>
       <y>180</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>JKL</string>
     </property>
    </widget>
    <widget class="QLabel" name="mno_label">
     <property name="geometry">
      <rect>
       <x>171</x>
       <y>180</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>MNO</string>
     </property>
    </widget>
    <widget class="QLabel" name="pqrs_label">
     <property name="geometry">
      <rect>
       <x>49</x>
       <y>240</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>PQRS</string>
     </property>
    </widget>
    <widget class="QLabel" name="tuv_label">
     <property name="geometry">
      <rect>
       <x>112</x>
       <y>240</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>TUV</string>
     </property>
    </widget>
    <widget class="QLabel" name="wxyz_label">
     <property name="geometry">
      <rect>
       <x>168</x>
       <y>240</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>WXYZ</string>
     </property>
    </widget>
    <widget class="QLabel" name="plus_label">
     <property name="geometry">
      <rect>
       <x>117</x>
       <y>300</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <zorder>plus_label</zorder>
    <zorder>wxyz_label</zorder>
    <zorder>tuv_label</zorder>
    <zorder>pqrs_label</zorder>
    <zorder>mno_label</zorder>
    <zorder>jkl_label</zorder>
    <zorder>ghi_label</zorder>
    <zorder>def_label</zorder>
    <zorder>abc_label</zorder>
    <zorder>pushButton_8</zorder>
    <zorder>pushButton_2</zorder>
    <zorder>pushButton_6</zorder>
    <zorder>pushButton_7</zorder>
    <zorder>pushButton_back</zorder>
    <zorder>pushButton_4</zorder>
    <zorder>phoneNum_lineEdit</zorder>
    <zorder>pushButton_0</zorder>
    <zorder>pushButton_asterisk</zorder>
    <zorder>pushButton_5</zorder>
    <zorder>pushButton_hashtag</zorder>
    <zorder>pushButton_3</zorder>
    <zorder>pushButton_call</zorder>
    <zorder>pushButton_9</zorder>
    <zorder>pushButton_1</zorder>
    <zorder>addNumber_label</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionOpen_an_Address_Book"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_an_Address_Book">
   <property name="text">
    <string>Open an Address Book...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
